### PR TITLE
[JAX] Add AbsMaxObserver

### PR DESCRIPTION
**Pull request by wpietka**
## Type of Change

feature

## Description

Implements new AbsMaxObserver for the JAX static quantization calibration path and wires it into eligible layers.

## Expected Behavior & Potential Risk

- MinMaxObserver is used for asymmetric int8 quantization
- AbsMaxObserver is used for every other case

## How has this PR been tested?

- New unit tests: test/jax/test_observers.py

## Dependency Change?

None

### CODE REVIEW

File: neural_compressor/jax/quantization/layers_static.py [hunk 1]
```diff
@@ -153,6 +153,115 @@ def is_calibrated(self):
         return not (jnp.isinf(self.min_val.value).any())
 
 
+class AbsMaxObserver(keras.layers.Layer):
+    """Observer that tracks the running maximum absolute value for calibration.
+
+    Unlike MinMaxObserver, this observer only records the maximum absolute value
```
Comment (anko-intel) [inline neural_compressor/jax/quantization/layers_static.py]: Don't compare to other, just describe what is for, also remove "small performance gain"

**Review by anko-intel — CHANGES_REQUESTED** [review]
see also:
 max_val = jnp.max(jnp.abs(orig_weight), axis=axis, keepdims=True)
in neural_compressor/jax/utils/utility.py
Maybe we can remove max(abs( there?

File: neural_compressor/jax/quantization/layers_static.py (hunk 1 above)
Comment (wpietka) [reply neural_compressor/jax/quantization/layers_static.py]: done

**Comment by wpietka** [comment]
> see also: max_val = jnp.max(jnp.abs(orig_weight), axis=axis, keepdims=True) in neural_compressor/jax/utils/utility.py Maybe we can remove max(abs( there?

get_scale() is used both for activations and kernel. While for activations it's redundant due to AbsMaxObserver, it is still necessary for kernel scale

**Review by anko-intel — APPROVED** [review]
LGTM

**Review by anko-intel — APPROVED** [review]
LGTM
